manifests/custom_config.pp

```diff
@@ -90,11 +90,12 @@
     $notifies = undef
   }
 
+  $_file_path = join([$confdir, $_filename], '/')
```

Comment:
I do wonder what the value of join here is over simple string interpolation. It's not like you win anything in compatibility (unlike OS specific join for Windows/Unix) and IMHO readability is also worse.

Reply:
Had used the join as I had simply copied from he remove_command array, but it does read easier as a string